dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -865,14 +755,15 @@ void ExchangeReceiverBase<RPCContext>::connectionDone(
         copy_live_conn);
 
     if (copy_live_conn == 0)
-    {
         LOG_DEBUG(log, "All threads end in ExchangeReceiver");
-    }
     else if (copy_live_conn < 0)
         throw Exception("live_connections should not be less than 0!");
```

Comment:
If exception triggered, the cv.notify_all() won't be called, is it ok?

Reply:
I think that cv.notify_all() must be triggered as it must be equal to 0 before it's less than 0.